lib/billing_facade_client.rb

```diff
@@ -39,12 +39,16 @@ def self.get_unit_price(cost_code, product_name)
     end
   end
 
+  def self.encode_for_url(str)
+    ERB::Util.url_encode(str)
```

Comment:
Can this be removed now that we're using underscores or dashes instead of slashes?